app/dashboard/models.py

```diff
@@ -2042,6 +2042,7 @@ def psave_bounty(sender, instance, **kwargs):
         'Months': 5,
     }
 
+    instance.github_url = instance.github_url.lower()
```

Comment:
github seems to be case insensitive  ( https://github.com/AbieFund/abie/issues/10 == https://github.com/abiefund/abie/issues/10 ) and this makes a big diff in db perf so seems like a little to give and a lot to gain to me